src/main/java/core/basesyntax/dao/Storage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> storage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the `storage` field should not be public. It's a bad practice to make your class fields exposed. Consider using a private modifier and providing access to this field via public methods.